interfaces/builtin/camera.go

```diff
@@ -36,6 +36,9 @@ const cameraConnectedPlugAppArmor = `
 # VideoCore cameras (shared device with VideoCore/EGL)
 /dev/vchiq rw,
 
+# Blackmagic cameras
+/dev/blackmagic/io[0-9]*,
```

Comment:
are these video subsystem devices? if not, then this will need associated udev rules, can you paste the output of `udevadm info --query property --name /dev/blackmagic/io*` here and we can help write the udev rules to add to the camera interface?

Reply:
@anonymouse64 
DEVPATH=/devices/pci0000:00/0000:00:04.0/0000:02:00.0/misc/blackmagic!io0
DEVNAME=/dev/blackmagic/io0
DEVMODE=0666
MAJOR=10
MINOR=58
SUBSYSTEM=misc

Reply:
so since this is not in the video subsystem this will need udev rules in addition to the existing rules for the video subsystem, thanks @otacke

Reply:
err rather to be more specific, it is not identified as a kernel `video*` device, it needs a different rule

Reply:
I've added the appropriate udev modificiation.